x/migration/recovery/recovery_allowlist.go

```diff
@@ -2059,13 +2061,33 @@ var recoveryAllowlist = []string{
 	"9ea35fee7695c2f639864ee38a5a448c20ae4547",
 	"319cf4360ba850d6c004910fbed197bde045fb9c",
 	"381add8cf814f45b8bf6dea56ac185c2358b8bf2",
+}
 
-	// Module accounts
+var moduleAccountsAllowlist = []string{
 	"dao",
 	"fee_collector",
 }
 
-// GetRecoveryAllowList returns the list of addresses that are allowed to be recovered
-func GetRecoveryAllowList() []string {
-	return recoveryAllowlist
+// TODO_FOLLOWUP: Add other list as the allowlist spreadsheet is updated
```

Comment:
No need for a TODO here. 

Just link to the [google docs](https://docs.google.com/spreadsheets/d/1V5Ge9lFXmX1A81DNZUOfYD6GSyuYrFwdXx4Wlhcnbck/edit?gid=1220065218#gid=1220065218) inline.

Please move this to the top of the file too.